playbooks/tasks

```diff
@@ -0,0 +1 @@
+../tasks
```

Comment:
this is unrelated to the new playbook and is rather a fix for an existing playbook so maybe just submit as a separate pull request.